File: ATS.UI/ClientApp/src/app/core/interfaces/note.ts
export class INoteDisplay {
    noteID: number;
    requirementID: number;
    requirement: string;
    candidateID: number;
    candidate: string;
    reviewQueueID: number;
    reviewQueue: string;
    reviewerID: number;
    reviewStatusID: number;
    reviewStatus: string;
    rateType?: string;
    rate?: number;
    reviewer: string;
    noteDesc: string;
    noteBy: number;
    dateNoted: Date;
    recordID?: number;
    dateCreated: Date;
    createdBy: string;
    dateModified: Date;
    modifiedBy: string;
    timeStamp: Date;
}

export class INoteForm {
    NoteID: number;
    RequirementID: number;
    CandidateID: number;
    ReviewQueueID: number;
    ReviewerID: number;
    ReviewStatusID: number;
    NoteDesc: string;
    NoteBy: number;
    DateNoted: Date;
    //recordID?: number;
    DateCreated: Date;
    CreatedBy: string;
    DateModified: Date;
    ModifiedBy: string;
   // TimeStamp: Date;
}

File: ATS.DTO/Note.cs
using System;

namespace ATS.DTO
{
    public class Note : AbstractDataObject
    {
        public int NoteID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        public int? RequirementID { get; set; }
        public int? CandidateID { get; set; }
        public int? ReviewQueueID { get; set; }
        public int? ReviewerID { get; set; }
        public int? ReviewStatusID { get; set; }
        public string NoteDesc { get; set; }
        public int NoteBy { get; set; }
        public DateTime? DateNoted { get; set; }
        [Ignore]
        public string Requirement { get; set; }
        [Ignore]
        public string Candidate { get; set; }
        [Ignore]
        public string NoteByName { get; set; }
        [Ignore]
        public string Reviewer { get; set; }
        [Ignore]
        public string ReviewStatus { get; set; }
    }
}